.github/actions/build-cl/action.yaml

```diff
@@ -8,7 +8,7 @@ runs:
       uses: actions/checkout@v5
       with:
         repository: smartcontractkit/chainlink
-        ref: becbfb2fc0774c97f7168780563ac9090aa3e3f1
+        ref: 585838629ce2be9fcf3f0ae9d79b72de6b892f08
```

Comment:
Is there a reason for bumping or just staying up to date w/ latest develop on chainlink?

Reply:
Actually this looks like a non-develop branch, if you can't merge the chainlink PR in time make sure to re-bump after a merge (and keep that branch un-deleted on remote)